taxonomies/vitamins.txt

```diff
@@ -1628,6 +1638,7 @@ sv:dexpantenol
 <en:vitamin A
 <en:vitamin D2
 fi:Vitamiini A- ja D2
```

Comment:
This entry should be completely removed in fact, it means "Vitamin A and D2", but we should not add combinations of vitamins in the taxonomy, it will be handled instead by the ingredients parser.

Reply:
Deleted it!

Reply:
Thank you!

I just noticed there are 2 other similar cases, could you delete them as well?

```
<en:vitamin A
<en:vitamin D2
fi:Vitamiini A- ja D2
nl:vitamine D2
<en:vitamin A
<en:vitamin D
da:vitamin A og D
ca:vitamines A i D, A i D
de:Vitamin A und D
es:vitaminas A y D, A y D
fi:A- ja D-vitamiini
fr:vitamines A et D, A et D
nl:vitamine A en D, A en D
sv:vitamin A och D, vitamin A och vitamin D
```